src/frontend/Komponenter/Behandling/Brev/BrevInnhold.tsx

```diff
@@ -76,6 +79,7 @@ type Props = {
     flyttAvsnittNed: (avsnittId: string) => void;
     context: FritekstBrevContext;
     behandlingsårsak?: Behandlingsårsak;
+    stønadstype?: Stønadstype;
```

Comment:
Tror ikke denne trenger å kunne være undefined. I behandlingscontext har vi stønadstype, og for frittstående brev kan vi heller hardkode overgangsstønad nå, og så må vi senere legge til støtte for å velge fagsak/stønadstype.

Reply:
Godt poeng, endret i c75102a